### PR TITLE
Add ssl error from weird misconfig relating to nabucasa cloud

### PR DESCRIPTION
**Pull request by spookyuser**

### CODE REVIEW

File: docs/troubleshooting/errors.md [hunk 1]
```diff
@@ -37,3 +37,4 @@ This occurs when the data sent during set up does not meet Home Assistant's expe
 This error is usually caused by one of the two following issues:
 *   You have denied local network access to the app. To solve the issue on iOS, open the Home Assistant entry in the system settings and verify that local network is enabled.
 *   You have configured an incorrect external url in your instance. E.g. when forwarding external port 443 to your instance's port (normally 8123), you don't have to append a port to your URL.
+* You are logged into nabu casa cloud without a subscription and trying to access your instance with a reverse proxy. [Just log out from your nabu casa cloud account.](https://community.home-assistant.io/t/ios-16-beta-companion-app-tries-to-connect-to-old-nabu-casa-url/440711/6)
```
Comment (dshokouhi) [inline docs/troubleshooting/errors.md:40]:
```suggestion
* You are logged into Nabu Casa cloud without a subscription and trying to access your instance with a reverse proxy. Just log out from your Nabu Casa cloud account.
```

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a small suggestion as we should make sure to capitalize the company name and we also do not need to link to a community post, just mentioning the issue and fix is good enough :)

**Comment by spookyuser** [comment]
Awesome, np!

**Review by dshokouhi — APPROVED** [review]
Thank you!

**Pull request merged**